scriptmodules/supplementary/pegasus-fe.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="pegasus-fe"
+rp_module_desc="Pegasus: A cross platform, customizable graphical frontend (latest alpha release)"
+rp_module_licence="GPL3+ https://raw.githubusercontent.com/mmatyas/pegasus-frontend/master/LICENSE.md"
+rp_module_section="exp"
+rp_module_flags="!mali frontend"
+
+function depends_pegasus-fe() {
+    local depends=(
+        curl
+        gstreamer1.0-alsa
+        gstreamer1.0-plugins-good
+        jq
+        libsdl2-dev
+        policykit-1
+    )
+    isPlatform "rpi" && depends+=(gstreamer1.0-omx)
+
+    getDepends "${depends[@]}"
+}
+
+function install_bin_pegasus-fe() {
+    # get all asset urls for the latest continuous release
+    local all_assets
+    all_assets="$(curl -s -XGET https://api.github.com/repos/mmatyas/pegasus-frontend/releases/tags/continuous)" || return
+    all_assets="$(echo "${all_assets}" | jq -r '.assets[] | .browser_download_url')"
+
```

Comment:
is it possible to use wget as we use it elsewhere for our downloading (saves a dependency) ?